src/main/java/ch/njol/skript/events/EvtClick.java

```diff
@@ -50,42 +33,42 @@ public class EvtClick extends SkriptEvent {
 	/**
 	 * Click types.
 	 */
-	private final static int RIGHT = 1, LEFT = 2, ANY = RIGHT | LEFT;
+	private static int RIGHT = 1, LEFT = 2, ANY = RIGHT | LEFT;
 
 	/**
 	 * Tracks PlayerInteractEvents to deduplicate them.
 	 */
-	public static final ClickEventTracker interactTracker = new ClickEventTracker(Skript.getInstance());
+	public static ClickEventTracker interactTracker = new ClickEventTracker(Skript.getInstance());
```

Comment:
The final can remain on fields. This is not related to the common final abuse of parameters seen in Skript coding standards.
```suggestion
	public final static ClickEventTracker interactTracker = new ClickEventTracker(Skript.getInstance());
```